.travis.yml

```diff
@@ -89,6 +89,14 @@ script:
   - if [ "$SAFETY_PROFILE" != "1" ]; then
       $DDS_ROOT/tests/cmake_integration/run_ci_tests.pl;
     fi
+  - if [ "$NO_TESTS" == "1" ]; then
+      echo Building Messenger;
+      source setenv.sh;
+      cd tests/DCPS/Messenger;
+      mwc.pl -type gnuace;
```

Comment:
You can combine this line with the previous one since `mwc.pl` takes a directory name as a command line argument.  It avoid the need to manage the `cd` state in different steps.

Reply:
This sequence was actually wrong. The correct one is:
      cd tests/DCPS/Messenger;
      mwc.pl -type gnuace .;
      make DDS_Messenger_Subscriber DDS_Messenger_Publisher;
      cd ../../..;

I am guessing I need to be in tests/DCPS/Messenger to correctly run nmake?

Reply:
I wouldn't do it that way, but maybe that's just me.  I don't like managing the current directory state separately, it's something else that can go wrong:
```
$ACE_ROOT/bin/mwc.pl -type gnuace tests/DCPS/Messenger;
make -C tests/DCPS/Messenger
```

Reply:
I agree. This is done now.